### DISCUSSION TITLE
Is there anything I can do to make dataset without metadata like CRS to merge, union with dataset that belongs to <class GeoDataset>??

### DISCUSSION THREAD

**Discussion by nam-drun** [torchgeo/torchgeo#2198] [Q&A]
Hi, I downloaded plenty RS dataset that doesn't have metadata at all so I wonder is there anyway to add metadata, or anyway to reproject without CRS? I have seen the python notebook and have a look at github issue/pull about <class NonGeoDataset> but it doesn't answer the issue i'm looking for.

**Comment by adamjstewart** [comment]
> Is there anything I can do to make dataset without metadata like CRS to merge, union with dataset that belongs to <class GeoDataset>??

The union is based on geographic bounding boxes. If you don't know the geographic bounding boxes of the data, it isn't possible to compute the union or intersection.

> I downloaded plenty RS dataset that doesn't have metadata at all so I wonder is there anyway to add metadata

If you have already downloaded images without geospatial metadata, it is technically possible to add metadata, but you have to first download this metadata. Honestly, it would be much easier to redownload all of the data from scratch.

> anyway to reproject without CRS?

If you don't know the initial CRS of the data, it is not possible to project it into a new CRS.
